aten/src/ATen/core/OpsAlreadyMovedToC10.cpp

```diff
@@ -0,0 +1,598 @@
+#include "OpsAlreadyMovedToC10.h"
+
+namespace at {
+
+// TODO Once all ATen ops are moved to c10, this file should be removed
+
+const std::unordered_set<c10::OperatorName>& aten_ops_already_moved_to_c10() {
```

Comment:
IIUC, @gchanan is trying to get rid of the checked in autogenerated files. So I'm pretty unhappy that we're adding another checked in autogenerated file in this patch. Is it *absolutely* necessary that this file lives in core? In particular, I find it hard to believe that mobile actually needs to depend on this file (and it really shouldn't, because you're going to pay for binary size to hold the unordered set and strings). If mobile doesn't depend on this file, there is no reason for the autogenerated file to be checked in.

Reply:
I've tried several other alternatives like adding this code to VariableType.cpp or TypeDefault.cpp before but they didn't work. ATenDispatch.h/cpp depends on this and is part of the ATen/core build and not having this file in ATen/core caused linker errors.

I don't really like this either but I think it's ok as an intermediate solution.

Reply:
My comment at https://github.com/pytorch/pytorch/pull/23668#issuecomment-530026140 is applicable here